WordPress/Classes/Utility/Analytics/WPAnalyticsTrackerAutomatticTracks.m

```diff
@@ -47,6 +47,7 @@ - (instancetype)init
     if (self) {
         _contextManager = [TracksContextManager new];
         _tracksService = [[TracksService alloc] initWithContextManager:_contextManager];
+        _tracksService.eventNamePrefix = [AppConstants eventNamePrefix];
```

Comment:
Since this is a property we can use dot notation: https://github.com/wordpress-mobile/objective-c-style-guide


```suggestion
        _tracksService.eventNamePrefix = AppConstants.eventNamePrefix;
```